src/mergeTranscendInputs.ts

```diff
@@ -0,0 +1,30 @@
+import { TranscendInput } from './codecs';
+import { getEntries } from '@transcend-io/type-utils';
+
+/**
+ * Combine a set of TranscendInput yaml files into a single yaml
+ *
+ * @param base - Base input
+ * @param inputs - The list of inputs
+ * @returns Merged input
+ */
+export function mergeTranscendInputs(
+  base: TranscendInput,
+  ...inputs: TranscendInput[]
+): TranscendInput {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const cloned: any = JSON.parse(JSON.stringify(base));
```

Comment:
Why not just use something like [`_.cloneDeep`](https://lodash.com/docs/4.17.15#cloneDeep)?

Reply:
just seems easier to do this